pkg/pipeline/transform/transform_network_test.go

```diff
@@ -313,3 +321,37 @@ func Test_Transform_AddIfScientificNotation(t *testing.T) {
 	require.Equal(t, true, output[0]["smaller_than_10_Evaluate"])
 	require.Equal(t, 1.2345e-67, output[0]["smaller_than_10"])
 }
+
+func Test_TransformNetworkOperationNameCustomServices(t *testing.T) {
+	rules := api.NetworkTransformRules{
+		api.NetworkTransformRule{
+			Input:      "dstPort",
+			Output:     "service",
+			Type:       "add_service",
+			Parameters: "protocol",
+		},
+	}
+
+	entry := test.GetIngestMockEntry(false)
+
+	var networkTransform = Network{
+		api.TransformNetwork{
+			Rules: rules,
+		},
+	}
+
+	err := netdb.Init("", "")
+	require.NoError(t, err)
+	output := networkTransform.Transform([]config.GenericMap{entry})
+	require.Equal(t, "ssh", output[0]["service"])
+
+	sshService := "custom_ssh             22/tcp                          # The Secure Shell (SSH) Protocol\n"
+	err = os.WriteFile("/tmp/services", []byte(sshService), 0644)
+	require.NoError(t, err)
+
+	err = netdb.Init("", "/tmp/services")
```

Comment:
This test is for non-empty services file. A similar test should be done for non-empty protocol file.

Reply:
Added another test for network protocols